PerformanceTests/TestBase.cs

```diff
@@ -38,18 +38,27 @@ private static void initializeRoslynForegroundThreadDataObject()
 		}	
 
 		[Setup]
-		public void Setup()
+		public virtual void Setup()
 		{
 			Host = new VSEmbed.DemoApp.EditorWindow();
 			Host.Show();
+			SetupHost();
+		}
+
+		public virtual void SetupHost()
```

Comment:
Does any test **not** override `SetupHost()`? Consider making it abstract if not.